### PR TITLE
[PLAYER-2647]Live button is not shown in app during playback of live assets

### PR DESCRIPTION
**Pull request by SergeyBicarte**
Fixed Live button for live assets, added red/gray circle along with position in DVR window in case it is not in current time.

### CODE REVIEW

File: sdk/react/controlBar.js [hunk 1]
```diff
@@ -172,12 +176,12 @@ class ControlBar extends Component {
         volumeControlColor: this.getVolumeControlColor(),
       },
       timeDuration: {
-        onPress: this.props.live ? this.props.live.onGoLive : null,
         playHeadTimeStyle: [styles.playheadLabel, {'fontSize': labelFontSize}],
         durationStyle: [styles.durationLabel, {'fontSize': labelFontSize}],
         completeTimeStyle: [styles.completeTimeStyle],
         playHeadTimeString: this.getPlayHeadTimeString(),
         iconTouchableStyle: styles.iconTouchable,
+        liveCircle: this.props.live ? (this.props.live.isLive ? styles.liveCircleActive : styles.liveCircleNonActive) : null,
```
Comment (atopilskii) [inline sdk/react/controlBar.js]: What do you thing about clean up this line?

File: sdk/react/controlBar.js [hunk 2]
```diff
@@ -59,7 +59,11 @@ class ControlBar extends Component {
 
   getDurationString = () => {
     if (this.props.live) {
-      return null;
+      if (this.props.live.isLive) {
```
Comment (atopilskii) [inline sdk/react/controlBar.js]: Could you check this line with linter? I see that we can destruct props.

File: sdk/react/controlBar.js (hunk 1 above)
Comment (atopilskii) [reply sdk/react/controlBar.js]: Could you check this line with linter? I see that we can destruct props.

File: sdk/react/panels/videoView.js [hunk 3]
```diff
@@ -78,22 +78,14 @@ class VideoView extends Component {
     if (this.props.live) {
       const isLive = this.props.playhead >= this.props.duration * VALUES.LIVE_THRESHOLD;
       return ({
-        label:
-          isLive ? Utils.localizedString(this.props.locale, 'LIVE', this.props.localizableStrings) :
-            Utils.localizedString(this.props.locale, 'GO LIVE', this.props.localizableStrings),
-        onGoLive: isLive ? null : this.onGoLive
+        label: Utils.localizedString(this.props.locale, 'LIVE', this.props.localizableStrings),
```
Comment (atopilskii) [inline sdk/react/panels/videoView.js]: Could you check this line with linter? I see that we can destruct props.

**Review by loginov-rocks — CHANGES_REQUESTED** [review]
Waiting for fixes suggested by @atopilskii

File: sdk/react/controlBar.js (hunk 1 above)
Comment (SergeyBicarte) [reply sdk/react/controlBar.js]: Done

File: sdk/react/controlBar.js (hunk 2 above)
Comment (SergeyBicarte) [reply sdk/react/controlBar.js]: Done

File: sdk/react/panels/videoView.js (hunk 3 above)
Comment (SergeyBicarte) [reply sdk/react/panels/videoView.js]: Done

**Review by atopilskii — APPROVED** [review]
Great work! Thanks!

File: sdk/react/controlBar.js [hunk 4]
```diff
@@ -50,19 +50,26 @@ class ControlBar extends Component {
...
-    } else {
-      return Utils.secondsToString(this.props.duration);
+    const {
+      live, duration, playhead,
+    } = this.props;
+    if (live) {
+      if (live.isLive) {
+        return null;
+      }
+      return (` -${Utils.secondsToString(duration - playhead)}`);
```
Comment (loginov-rocks) [inline sdk/react/controlBar.js:70]: Should that be just `Utils.secondsToString(duration - playhead)`?

Comment (SergeyBicarte) [reply sdk/react/controlBar.js:70]: Probably you are right , but change it to Utils.secondsToString(playhead - duration)

Comment (SergeyBicarte) [reply sdk/react/controlBar.js:70]: After some checking, that code add two symbols at start of string : space and minus, don't see we could improve that place somehow

**Review by AlinaVoronkovaEpam — APPROVED** [review]

**Pull request merged**